libs/types/src/smartcard_auth/dipped_smartcard_auth.ts

```diff
@@ -26,7 +32,7 @@ export interface RemoveCard {
 export interface SuperadminLoggedIn {
   readonly status: 'logged_in';
   readonly user: SuperadminUser;
-  readonly card?: CardStorage;
+  readonly card?: CardStorage & CardProgramming;
```

Comment:
Cool that this approach of having card functionality "mixins" actually worked out in practice!